scripts/qa_crowbarsetup.sh

```diff
@@ -4786,6 +4786,20 @@ function onadmin_allow_vendor_change_at_nodes
     done
 }
 
+function crowbar_nodeupgrade_finished
+{
+    if grep current_step $upgrade_progress_file | grep -v nodes ; then
+        echo "'nodes' step finished successfuly"
+        return 0
+    fi
+
+    if grep -q failed $upgrade_progress_file ; then
```

Comment:
doesn't it make sense to check for this first before exiting on the lack of nodes (see above) ?